cloudferrylib/os/image/glance_image.py

```diff
@@ -136,6 +201,19 @@ def read_info(self, **kwargs):
 
         info = {'images': {}}
 
+        def image_valid(img, date):
```

Comment:
Ok, but I guess we are making code more complex with such nested things. From my point of view it's better to have private methods in class in such cases (with `def _method_name`). But I am not sure about best practices in such situations) So, whatever you like :)